crates/sui-core/src/execution_cache/writeback_cache.rs

```diff
@@ -1258,18 +1344,26 @@ impl TransactionCacheRead for WritebackCache {
         do_fallback_lookup(
             digests,
             |digest| {
-                Ok(
-                    if let Some(tx) = self.dirty.pending_transaction_writes.get(digest) {
-                        CacheResult::Hit(Some(tx.transaction.clone()))
-                    } else if let Some(tx) = self.cached.transactions.get(digest) {
-                        CacheResult::Hit(Some(tx.clone()))
-                    } else {
-                        CacheResult::Miss
-                    },
-                )
+                self.metrics
+                    .record_cache_request("transaction_block", "uncommitted");
+                if let Some(tx) = self.dirty.pending_transaction_writes.get(digest) {
+                    return Ok(CacheResult::Hit(Some(tx.transaction.clone())));
```

Comment:
Should we record `hit` here? (also in below functions multi_get_executed_effects_digests and multi_get_effects?

Reply:
good catch!